client/src/components/LibraryNavigator/LibraryModel.ts

```diff
@@ -27,7 +26,7 @@ class LibraryModel {
     await window.withProgress(
       {
         location: ProgressLocation.Notification,
-        title: "Connecting to SAS session...",
+        title: l10n.t("Connecting to SAS session..."),
```

Comment:
These don't follow our typical format in `package.nls.json`. Do we need to update all of these strings to conform to that format?